.storybook/assets/404.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Page not found</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <link rel="icon" type="image/svg+xml" href="./favicon.svg">
+    <style>
+      body {
+        --spectrum-background-layer-1-color: rgb(253, 253, 253);
+
+        margin: 0;
+        padding: 0;
+        block-size: 100vh;
+        background-color: var(--spectrum-background-layer-1-color);
+      }
+
+      main {
+        --spectrum-neutral-content-color-default: rgb(34, 34, 34);
+
+        color: var(--spectrum-neutral-content-color-default);
+        display: flex;
+        padding-inline: 20px;
+        padding-block: 4rem;
+        flex-direction: column;
+        align-items: flex-start;
+        gap: 60px;
+        max-width: 1000px;
+        margin: auto;
+      }
+
+      img {
+        max-inline-size: 100%;
+        margin-block: -20px 20px;
+      }
+
+      h1, h2, p {
+        --spectrum-sans-font-family-stack: "Nunito Sans",-apple-system,".SFNSText-Regular","San Francisco",BlinkMacSystemFont,"Segoe UI","Helvetica Neue",Helvetica,Arial,sans-serif;
+
+        -webkit-font-smoothing: antialiased;
+        font-family: var(--spectrum-sans-font-family-stack);
+      }
+
+      h1 {
+        font-weight: 700;
+        font-size: 32px;
+        margin-block: 20px 8px;
+      }
+
+      h2 {
+        font-size: 24px;
+        margin-block: 20px 8px;
+        padding-block-end: 4px;
+      }
+
+      p {
+        font-size: 14px;
+        line-height: 24px;
+        margin-block: 16px;
+      }
+
+    </style>
+  </head>
+  <body>
+    <main>
+      <h1>404</h1>
+      <section>
+        <img src="/spectrum_illustration_2x.png" alt="">
+        <h2>It's not you. It's us.</h2>
+        <p>We've made lots of changes recently to our documentation site. In preparation for Spectrum 2, we've been hard at work migrating our documentation here, to Storybook.</p>
```

Comment:
```suggestion
        <p>We've made lots of changes to the Spectrum CSS documentation site, including consolidating all of the documentation and moving it to a single location within Storybook.</p>
```
A suggestion based on Rise's note about this needing revisiting in the future, so this content does not get stale as quickly.

Reply:
great idea!